examples/compiled/base64.sh

```diff
@@ -46,7 +46,7 @@ unpack_escaped_string() {
         __buf="${__buf#?}" # Remove the current char from $__buf
         ;;
       *)
-        __c=$(LC_CTYPE=C printf "%d" "'${__buf%"${__buf#?}"}")
+        __c=$(LC_CTYPE=C printf "%d" "'${__buf%"${__buf#?}"}"); __c=$((__c > 0 ? __c : 256 + __c))
```

Comment:
Bash-2.05a uses `signed char` so everything in the extended range gets encoded as a negative number. We could remove this line if we choose to not support early versions of bash, but it worth the extract few characters so we can say we support a version of bash from 2001.